.github/workflows/release-prepare.yml

```diff
@@ -259,11 +259,11 @@ jobs:
 
           cd "${HOLOCHAIN_REPO}"
 
-          git tag --sort=-taggerdate | grep holochain-
+          git tag --sort=-taggerdate --merged $(git branch --show-current) | grep holochain-
 
-          export LATEST_HOLOCHAIN_TAG=$(git tag --sort=-taggerdate | grep holochain- | head -n1)
-          export LATEST_HOLOCHAIN_VERSION=${LATEST_HOLOCHAIN_TAG/holochain-/}
           export RELEASE_BRANCH=$(git branch --show-current)
```

Comment:
```suggestion
```

Reply:
I'm not sure what this suggestion should do any more because I committed the ones above and this no longer seems to make sense?

Reply:
it's added (as a duplicate) in line 262 (new count) so it can be removed here.